src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task10.java

```diff
@@ -37,6 +37,12 @@ public static void main(String[] args) {
     static int getNumberStation(int firstFuelReserves, int secondFuelReserves, int thirdFuelReserves) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        if (firstFuelReserves <= secondFuelReserves && firstFuelReserves <= thirdFuelReserves) {
+            return 1;
+        } else if (secondFuelReserves <= firstFuelReserves && secondFuelReserves <= thirdFuelReserves) {
+            return 2;
+        }else if (thirdFuelReserves <= secondFuelReserves) {
+            return 3;
+        }else return 1;
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"